platforms/platform-googleassistant/src/GoogleAssistantDevice.ts

```diff
@@ -3,8 +3,8 @@ import { GoogleAssistant } from './GoogleAssistant';
 import { Capability as NativeCapability } from './output';
 
 export enum GoogleAssistantCapability {
-  InteractiveCanvas = 'GOOGLE_ASSISTANT:INTERACTIVE_CANVAS',
```

Comment:
Could you elaborate why you removed the ´GOOGLE_ASSISTANT` prefix? This is a convention we use for platform specific capabilities: https://www.jovo.tech/docs/device#platform-specific-device-features